java/test/jmri/jmrix/loconet/LnOpsModeProgrammerTest.java

```diff
@@ -466,6 +466,78 @@ public void testOpsReadLocoNetMode() throws ProgrammerException {
 
      }
 
+     @Test
+     public void testOpsReadLocoNetModeLACKRejected() throws ProgrammerException {
+        // allow transponding
+        sm.setTranspondingAvailable(false);
+
+        LnOpsModeProgrammer lnopsmodeprogrammer = new LnOpsModeProgrammer(memo, 4, true);
+
+        lnopsmodeprogrammer.setMode(LnProgrammerManager.LOCONETOPSBOARD);
+        lnopsmodeprogrammer.readCV("12", pl);
+
+        // should have written
+        Assert.assertEquals("one message sent", 1, lnis.outbound.size());
+        Assert.assertEquals("No programming reply", 0, pl.getRcvdInvoked());
+
+        Assert.assertEquals("sent", "EF 0E 7C 2F 00 00 04 00 00 0B 00 7F 7F 00", lnis.outbound.get(0).toString());
```

Comment:
It is helpful if the assertion message is more meaningful than simply "sent", to help a test-fail-debugger to find the correct test.  I usually give unique messages, perhaps in this case something like "first CV12 read programming request".